maintenance/MaintenanceReferenceOntology.rdf

```diff
@@ -168,15 +177,15 @@
 				<owl:someValuesFrom rdf:resource="&iof-maint;FailedState"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>failure event</rdfs:label>
+		<rdfs:label xml:lang="en-US">failure event</rdfs:label>
 		<rdfs:seeAlso>ISO81346 for examples of functions</rdfs:seeAlso>
 		<skos:example>explosion, seizure, loss of power, loss of control</skos:example>
-		<iof-av:explanatoryNote>the function can be a combination of functions</iof-av:explanatoryNote>
-		<iof-av:firstOrderLogicDefinition>FailureEvent(e) ↔ Event(e) ∧ ∃o(FailedState(o) ∧  initiates(e, o)) ∧ ∃i,f,p1(MaintainableMaterialItem(i) ∧ hasParticipantAtAllTimes(e, i) ∧ PrimaryFunction(f) ∧ hasFunction(i, f) ∧ FunctioningProcess(p1) ∧ realizes(p1, f) ∧ precedes(p1, e) ∧ ¬∃p2(FunctioningProcess(p2) ∧ realizes(p2,f) ∧  precedes(p1, p2) ∧ precedes(p2, e)))</iof-av:firstOrderLogicDefinition>
+		<iof-av:explanatoryNote>the event can be the loss of the primary function or a combination of functions</iof-av:explanatoryNote>
+		<iof-av:firstOrderLogicDefinition>FailureEvent(e) ↔ Event(e) ∧ ∃o(FailedState(o) ∧ initiates(e, o)) ∧ ∃i,f,p1(MaintainableMaterialItem(i) ∧ hasParticipantAtAllTimes(e, i) ∧ PrimaryFunction(f) ∧ hasFunction(i, f) ∧ FunctioningProcess(p1) ∧ realizes(p1, f) ∧ precedes(p1, e) ∧ ¬∃p2(FunctioningProcess(p2) ∧ realizes(p2,f) ∧ precedes(p1, p2) ∧ precedes(p2, e)))</iof-av:firstOrderLogicDefinition>
 		<iof-av:isPrimitive rdf:datatype="&xsd;boolean">false</iof-av:isPrimitive>
-		<iof-av:maturity>Provisional</iof-av:maturity>
-		<iof-av:naturalLanguageDefinition>event that causes an item to lose its ability to perform a required function</iof-av:naturalLanguageDefinition>
-		<iof-av:semiFormalNaturalLanguageDefinition>&apos;failure event&apos; : every instance of &apos;failure event&apos; is defined as exactly an instance of &apos;event&apos; e that &apos;initiates&apos; some &apos;failed state&apos; and that &apos;has participant at all times&apos; some &apos;maintainable material item&apos; that &apos;has function&apos; some &apos;primary function &apos; f that &apos;has realization&apos; some &apos;functioning process&apos; p1 that &apos;precedes&apos; e and there is no &apos;functioning process&apos; p2 such that p2 &apos;realizes&apos; f and p1 &apos;precedes&apos; p2 and p2 &apos;precedes&apos; e</iof-av:semiFormalNaturalLanguageDefinition>
+		<iof-av:maturity rdf:resource="&iof-av;Provisional"/>
```

Comment:
This is not necessary since the ontology is marked as provisional. Same for the rest of the classes.